server/src/main/java/com/objectcomputing/checkins/services/skills/Skill.java

```diff
@@ -14,9 +14,21 @@
 import java.util.UUID;
 
 @Entity
-@Table(name ="skills")
+@Table(name = "skills")
 public class Skill {
 
+    @Id
+    @Column(name = "skillid")
+    @AutoPopulated
+    @TypeDef(type = DataType.STRING)
+    private UUID skillid;
```

Comment:
I think we want it consistent and call it a `id` everywhere. we have #208 open for that purpose.

Reply:
If this gets changed, please make the appropriate change in the frontend too, it is looking for that skillid. That would have to be changed to id as well.

This is likely true for any api change going forward.

Reply:
done